Mention exceptions prominently with interfaces as non-upgradeable (#804)
digital-asset/docs.daml.com: docs/2.10.0/docs/concepts/ledger-model/upgrade-model-in-depth.rst

            data Demo = Demo with
              field1 : C T

Interface and Exception Definitions
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Both interface definitions and exception definitions cannot be upgraded. We

Reviewer: ```suggestion
Neither interface definitions nor exception definitions can be upgraded. We
```
